apps/chat-e2e/src/tests/sharedFilesAttachments.test.ts

```diff
@@ -0,0 +1,477 @@
+    'Arrow icon appears for the folder and file with the special chars in their names.\n' +
+    'Error message appears if to Share the conversation with an attachment from Shared with me\n' +
+    'Arrow icon stays for the file if the chat was unshared by the owner\n' +
+    'Arrow icon stays for the file if the chat was renamed or deleted, or model was changed\n' +
+    'Arrow icon disappears if all the users delete the file from "Shared with me"',
+  async ({
+    setTestIds,
+    conversationData,
+    dataInjector,
+    fileApiHelper,
+    mainUserShareApiHelper,
+    additionalUserShareApiHelper,
+    dialHomePage,
+    attachedFilesAssertion,
+    chatBar,
+    attachedAllFiles,
+    localStorageManager,
+    additionalShareUserSendMessage,
+    additionalShareUserConversations,
+    additionalShareUserLocalStorageManager,
+    additionalShareUserErrorToast,
+    additionalShareUserChat,
+    additionalShareUserConversationDropdownMenu,
+    additionalShareUserConversationData,
+    additionalShareUserAttachmentDropdownMenu,
+    additionalShareUserDialHomePage,
+    additionalShareUserAttachFilesModal,
+    additionalShareUserShareErrorToastAssertion,
+    additionalShareUserDataInjector,
+    conversations,
+    attachmentDropdownMenu,
+    attachFilesModal,
+    confirmationDialog,
+    conversationDropdownMenu,
+    chatHeader,
+    talkToSelector,
+    marketplacePage,
+    chat,
+    additionalSecondUserShareApiHelper,
+    sendMessage,
+    additionalSecondShareUserFileApiHelper,
+    additionalShareUserFileApiHelper,
+  }) => {
+    dialSharedWithMeTest.slow();
+    setTestIds(
+      'EPMRTC-4133',
+      'EPMRTC-4134',
+      /*'EPMRTC-4135,'*/
+      'EPMRTC-4155',
+      'EPMRTC-4123',
+      'EPMRTC-3116',
+      'EPMRTC-3122',
+      'EPMRTC-4164',
+    );
+    let imageUrl: string;
+    let imageUrl2: string;
+    let imageInConversationInFolderUrl: string;
+    let specialCharsImageUrl: string;
+    //TODO EPMRTC-4135 blocked by the #1076
+    // let imageInFolderUrl2: string;
+    let shareByLinkResponse: ShareByLinkResponseModel;
+    let shareFolderByLinkResponse: ShareByLinkResponseModel;
+    let defaultModel: ModelIds;
+    let conversationInFolder: Conversation;
+    //TODO EPMRTC-4135 blocked by the #1076
+    // let conversationToMove: Conversation;
+    const folderName = 'Folder with conversation';
+    const specialCharsFolder = `Folder ${ExpectedConstants.allowedSpecialChars}`;
+    let conversationWithSpecialChars: Conversation;
+
+    let conversationWithTwoResponses: Conversation;
+
+    await localStorageManager.setChatCollapsedSection(
+      CollapsedSections.Organization,
+    );
+
+    await dialTest.step(
+      'Upload image file to a conversation and prepare conversation with attachments in response',
+      async () => {
+        defaultModel = ModelIds.GPT_4_O;
+        await fileApiHelper.deleteAllFiles();
```

Comment:
this call is not necessary. The cleanup implemented in the `beforeTestCleanup` fixture is invoked for every test

Reply:
done